docs/ResultsList.md

```diff
@@ -0,0 +1,45 @@
+# Search Bar
+
+## Description
+
+`ResultsList` is a VTEX Component that represents the autocomplete from the search bar. This component can be imported and used by any VTEX App.
```

Comment:
Do you think `AutocompleteResults` would be a better name for this component?

Reply:
`AutocompleteResults` looks better to me.
Done!